tests/ColorLinesTest.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+namespace PHP_Parallel_Lint\PhpConsoleHighlighter\Test;
+
+use PHP_Parallel_Lint\PhpConsoleColor\ConsoleColor;
+use PHP_Parallel_Lint\PhpConsoleHighlighter\Highlighter;
+use ReflectionMethod;
+
+/**
+ * @covers PHP_Parallel_Lint\PhpConsoleHighlighter\Highlighter::colorLines
+ */
+class ColorLinesTest extends HighlighterTestCase
+{
+    /** @var string */
+    private $input = array(
+        7 => array (
+            0 => array (
```

Comment:
Are necessary array indexes from 0?

Reply:
Necessary: no, but as I'm mocking the input send to the function being tested, I kind of felt it would be good to be explicit about what the function expects to receive and what I'm sending it.